### ISSUE TITLE
RFE: Remove tokens/keys in log output

### ISSUE THREAD

**Issue by TJM** [larderdev/kubewise#33]
The logs should not contain authentication tokens or keys. For example:

```
kubewise-64db798b7f-m9chj:kubewise 2020/03/25 17:17:02 Error making request to Google Hangouts Chat Post "https://chat.googleapis.com/v1/spaces/xxxxxxxx/messages?key=(CENSORED)&token=(CENSORED)": dial tcp 216.58.193.74:443: connect: connection refused
```

**Comment by dtuite** [comment]
@TJM I just merged some code to take any tokens out of the logs. I also audited all the other points in the code where logging occurs to ensure this isn't a risk going forward. I'm about to release a new version with this change in it. Will let you know when that is live.

**Comment by TJM** [comment]
Thanks!

**Comment by dtuite** [comment]
Fixed in version 0.6.2.

```
helm repo update
helm install kubewise larder/kubewise --version 0.6.2 --blah blah
```
